interp/src/values.rs

```diff
@@ -186,9 +238,12 @@ impl Value {
 impl Into<u64> for Value {
     fn into(self) -> u64 {
```

Comment:
Silly question, but could this just call `self.as_u64()` instead of having another copy of the implementation?

Reply:
+1 to this as well, doesn't seem like we need to redefine the impl here